### ISSUE TITLE
Image list navigation hints/arrows and possible gallery on the bottom

### ISSUE THREAD

**Issue by vitorhugods** [stfalcon-studio/StfalconImageViewer#41]
**Is your feature request related to a problem? Please describe.**
When displaying a list of images in fullscreen, It seems a lot of users don't notice that you can scroll to the next image. They close the fullscreen viewer and tap on the next image. 

**Describe the solution you'd like**
A couple of new features I would like to discuss, so I can give a go on them in a PR:
* Possibility to add arrows buttons to the left/right of the screen, to be able to navigate to the next or previous image. It doesn't seem clear at the moment that this is a possibility. Maybe even enable previous/next navigation as in the infamous "Story" features in social media apps: tapping on the right goes to the next picture, tapping on the left goes to the previous one.
* Small horizontal list, containing thumbnails of the images. Just like a lot of gallery apps. User can scroll the thumbnails list, tap on an image to see it fullscreen.

Both of these could solve the issue. And I'm willing to give it a go, if this is wanted in the library.

** Alternatives **
I was thinking of just adding a view overlay in my project and move on. 
But I would be happy to share my implementation if it is wanted. So I want to discuss in here first.

Otherwise I can do it just for me, no problem.

**Comment by YashMundra2411** [comment]
Yes, I want the implementation to have small horizontal list containing thumbnail of all the images.

**Comment by lalitjadav007** [comment]
I needed too
